src/server/utilities/attributeFunctions/index.js

```diff
@@ -7,8 +7,9 @@ export const getScriptAttributes = bundleType => chunk => {
     const { url } = chunk;
 
     return {
-      crossOrigin,
+      async: false,
```

Comment:
Sets this explicitly to `false` as otherwise it set to `true`. 

Per React 19 https://react.dev/reference/react-dom/components/script#special-rendering-behavior, React takes over rendering of `script` tags under certain conditions, hoisting them to the `head` if `async:true` and `src` is set. This appears to causes issues for us, resulting in hydration errors occasionally and unpredictably happening.

Reply:
This special rendering behaviour sort of replaces what React Helmet does for us... right? Not for now, but if we think that's true, should we create a tech ticket to investigate whether we could remove React Helmet and use the default React behaviour 😬 ? OR if we moved everything to NextJS, would it mean that we wouldn't need to worry about this?

Reply:
I think we can replace it in _some_ instances, but not all: https://react.dev/blog/2024/12/05/react-19#support-for-metadata-tags

React suggests the likes of react-helmet is still useful for more complex things. Although its definitely worth exploring.

Reply:
Next.js has its own implementation of this with `next/head`, so we should move to that first perhaps.